lib/header.js

```diff
@@ -24,13 +23,22 @@ module.exports = (props) => {
     }
   };
   return yo`
-    <header style=${css(style)}>
-      <h1>All Dats</h1>
-      ${button({
-        text: 'Create new Dat',
-        click: props.create
-      })}
-      <input type="text" placeholder="Import dat" onkeydown=${keydown}>
+    <header style=${css(style)} class="dat-header">
```

Comment:
i'll remove this inline styles logic after we merge this. the idea was to replicate what react folks are doing (see https://speakerdeck.com/vjeux/react-css-in-js). ideally we'll eventually come up with a way to merge both appraoches, giving you the flexibility of sass _and_ all the other merits from the css-in-js idea